github-pr

```diff
@@ -12,11 +12,17 @@ def _check_required_fields(required, **args):
             sys.exit(1)
 
 
+def _return_status(commits):
+    for commit in commits:
+        for commit_status in commit.get_statuses():
+            return commit_status.state
```

Comment:
this should only have one return. probably only want to check the most recent status of the most recent commit. It's not clear if this is effectively doing that